File: Cpp/inheritance/Karyawan.cpp
using namespace std;

class Karyawan : public Manusia {
    private:
        string nomor_pegawai;
        string jabatan;
        string departemen;
    public:

    Karyawan(){
    }

    void setNomorPegawai(string nomor){
        this->nomor_pegawai = nomor;
    }

    string getNomorPegawai(){
        return nomor_pegawai;
    }

    void setJabatan(string jabatan){
        this->jabatan = jabatan;
    }

    string getJabatan(){
        return jabatan;
    }

    void setDepartemen(string departemen){
        this->departemen = departemen;
    }

    string getDepartemen(){
        return departemen;
    }

    ~Karyawan(){
    }
};

File: python/inheritance/Karyawan.py
from Manusia import Manusia

class Karyawan(Manusia):
    
    def __init__(self):
        self.no_pegawai = ""
        self.jabatan = ""
        self.department = ""
    
    def set_no_pegawai(self, no_pegawai):
        self.no_pegawai = no_pegawai
    
    def get_no_pegawai(self):
        return self.no_pegawai
    
    def set_jabatan(self, jabatan):
        self.jabatan = jabatan
    
    def get_jabatan(self):
        return self.jabatan
    
    def set_department(self, department):
        self.department = department
    
    def get_department(self):
        return self.department